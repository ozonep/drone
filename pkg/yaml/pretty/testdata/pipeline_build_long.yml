---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: test_build
  build:
    image: octocat/hello-world
    context: .
    args:
      foo: bar
      baz: boo
    labels:
      qux: qoo
    cache_from:
    - alpine
    - golang
