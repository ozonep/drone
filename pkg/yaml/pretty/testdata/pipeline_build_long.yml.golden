---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: test_build
  build:
    image: octocat/hello-world
    args:
      baz: boo
      foo: bar
    cache_from:
    - alpine
    - golang
    context: .
    labels:
      qux: qoo

...
